packages/front-end/pages/404.tsx

```diff
@@ -2,6 +2,7 @@ import * as Sentry from "@sentry/react";
 import { useRouter } from "next/router";
 import { useEffect } from "react";
 import { isSentryEnabled } from "@/services/env";
+import "../styles/tailwind.css";
```

Comment:
I think this page is rendered inside `_app`, so we don't need to import the css file here

Reply:
this was a test